examples/launcher/src/components/Dom.jsx

```diff
@@ -99,7 +97,6 @@ class Dom extends React.Component {
     return (
       <div className="Dom">
         <DomList root={this.state.root} selectEl={this.state.selectEl} onClick={el => this.onClick(el)} />
```

Comment:
If the intent is to re-add `DomDetail` later, perhaps just comment it?

Reply:
29588b7